Templates/AppSource App/.github/workflows/UpdateGitHubGoSystemFiles.yaml

```diff
@@ -30,10 +34,53 @@ env:
   ALGoRepoSettings: ${{ vars.ALGoRepoSettings }}
 
 jobs:
+  Initialize:
+    runs-on: windows-latest
+    name: Initialize
+    outputs:
+      UpdateBranches: ${{ steps.GetBranches.outputs.Result }}
+      TemplateUrl: ${{ steps.DetermineTemplateUrl.outputs.TemplateUrl }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - name: Read settings
+        id: ReadSettings
+        uses: microsoft/AL-Go-Actions/ReadSettings@main
+        with:
+          shell: powershell
+          get: templateUrl
+
+      - name: Get Workflow Multi-Run Branches
+        id: GetBranches
+        uses: microsoft/AL-Go-Actions/GetWorkflowMultiRunBranches@main
+        with:
+          shell: powershell
+          includeBranches: ${{ github.event.inputs.includeBranches }}
+
+      - name: Determine Template URL
```

Comment:
Could the determine template url somehow be included in the upper action?
Maybe a generic GetWorkflowInput, which takes parameters from input or from settings

Reply:
The new action, `GetWorkflowMultiRunBranches` is only to determine the branches when we want to run a workflow on multiple branches.
_TemplateURL_ is very specific for "Update AL-Go System Files", so that's why I kept it separate.